usagov/vote-gov-drupal
# VOTE-1033: Update Drupal Core from 10.0.1 to 10.0.7

## Jira ticket (required)
[VOTE-1033](https://bixal-projects.atlassian.net/browse/VOTE-1033)

## Description (optional)
Update Drupal Core to 10.0.7

## Deployment and testing (required, if applicable)
### Testing steps
1. After composer install && drush updb, go to Reports > Status Report and ensure Drupal Core is version 10.0.7
